src/NumericalAlgorithms/SphericalHarmonics/Python/Strahlkorper.cpp

```diff
@@ -35,5 +36,19 @@ void bind_strahlkorper(pybind11::module& m) {  // NOLINT
                              &ylm::Strahlkorper<Frame::Inertial>::l_max)
       .def_property_readonly("m_max",
                              &ylm::Strahlkorper<Frame::Inertial>::m_max);
+  m.def(
+      "ylm_legend_and_data",
```

Comment:
#5921 should help with the dependencies of this

Reply:
Thanks, rebased in this and added bindings for both reading and writing

Reply:
Are the following functions maybe better placed in the `StrahlkorperFunctions.cpp` bindings file since they do things with Strahlkorper data?

Reply:
They do Strahlkorper IO, so I think this is a good place for them.